src/main/src/main/java/org/geoserver/catalog/impl/AttributeTypeInfoImpl.java

```diff
@@ -152,6 +153,16 @@ public void setSource(String source) {
         this.source = source;
     }
 
+    @Override
+    public String getDescription() {
```

Comment:
Prefer if this was InternationalString right from the start; indicates human readable text.